openstack/CleanWlmDatabase/snapshot_delete.py

```diff
@@ -0,0 +1,55 @@
+import os
+import datetime
+from lib.utility import (setup_logger,
+                         file_manager,
+                         connect_db
+                         )
+
+
+def delete_snapshots(curr, conn, snapshots_ids):
+    for snapshot_id in snapshot_ids:
+        try:
+            snapshot_exists = "SELECT EXISTS(SELECT * FROM snapshots WHERE id = %s AND deleted = 1 AND status = \"deleted\")"

```

Comment:
can't we fetch all the given snaps at once, something like SELECT * FROM snapshots WHERE id in (<list_of_snap_ids>) AND deleted = 1 AND status = \"deleted\"